core/src/worker/mod.rs

```diff
@@ -262,8 +265,11 @@ impl Worker {
                     let sticky_metrics = metrics.with_new_attrs([workflow_sticky_poller()]);
                     new_workflow_task_buffer(
                         client.clone(),
-                        sqn.clone(),
-                        true,
+                        TaskQueue {
+                            name: sqn.clone(),
```

Comment:
Looks like we still embed the "normal" name in the sticky queue name, no need for that anymore (especially if the server supports build id based versioning and is aware of the normal_name field).

Reply:
Huh? That's explicitly when we _do_ need it -- otherwise why does the field even exist at all? We added normal name for this purpose.

Reply:
I mean this: https://github.com/temporalio/sdk-core/blob/31099415d28f983350d3cf1d005705de037ccb8d/core/src/lib.rs#L152

Reply:
Oooooh, that. Yeah... I suppose that should be fine to change.